build.sbt

```diff
@@ -0,0 +1,41 @@
+lazy val supportedScalaVersions = List("2.13.4", "2.12.12")
+
+ThisBuild / organization := "org.tree_annotation"
+ThisBuild / scalaVersion := "2.12.12"
+ThisBuild / name := "tao-parser"
+ThisBuild / description := "first truly universal syntax for structural communication at any scale, born of a vision of total intercommunication of all software systems"
+ThisBuild / licenses := List("MIT" -> url("https://choosealicense.com/licenses/mit/"))
+ThisBuild / homepage := Some(url("https://www.tree-annotation.org/"))
+ThisBuild / scmInfo := Some(ScmInfo(url("https://github.com/tree-annotation/tao"), "https://github.com/tree-annotation/tao.git"))
+
+ThisBuild / developers := List(Developer(
+  "Dariusz Jędrzejczak",
+  "Dariusz Jędrzejczak",
+  "jdwork@o2.pl",

```

Comment:
email will be truman at tree-annotation.org
can it be given a bit obfuscated like "truman" + "!@#".charAt(1) + "tree-annotation.org" to avoid spam?

Reply:
We can try